utilities/translation/tfcomp.js

```diff
@@ -31,6 +31,8 @@ let frFileMissList = [];
 
 const jsonFiles = buildJsonFileArray();
 
+const notSwapFile = (fname) => !fname.includes('.swp');
```

Comment:
Nice fix 👍 . Truly professional swap file abusers end up with .sw* to the ends of the earth. This should work well though.